### ISSUE TITLE
Replace filtered characters with per-character placeholder

### ISSUE THREAD

**Issue by ufna** [alexzel/bad-words-next#15]
It'll be cool to have ability to filter like this:

MYBADWORD -> *********
MYWORD -> ******

**Comment by alexzel** [comment]
Thanks for the feature request. I have implemented it in [v2.2.0](https://github.com/alexzel/bad-words-next/blob/main/CHANGELOG.md#220-2023-06-21) with [placeholderMode](https://github.com/alexzel/bad-words-next/wiki/Options#placeholdermode) option that you can change to `repeat` and make the placeholder be just one character like  `*` so it will achieve the behavior you describe above.

```js
const badwords = new BadWordsNext({
  data: en,
  placeholder: '*',
  placeholderMode: 'repeat'
})
...
```

**Comment by ufna** [comment]
@alexzel works like a charm, thank you!
